src/components/Navbar.js

```diff
@@ -10,8 +10,8 @@ export default (props) => {
       <Nav className='mr-auto'></Nav>
       <Nav style={{ alignItems: 'center' }}>
         <Nav.Link style={{ color: '#30323D' }} href="login">Log In</Nav.Link>
-        <Nav.Link style={{ color: '#30323D' }} href="signup">
-          Sign Up
+        <Nav.Link style={{ color: '#30323D' }} href="register">
+          Register
```

Comment:
Doesn't match the designs. I can't remember, did the group decide on language for this?